### PR TITLE
Dblatcher/readme

### PR DESCRIPTION
**Pull request by dblatcher**
## What does this change?
Updates README files.

Please do comment if anything looks wrong or there are missing topics.

### CODE REVIEW

File: libs/state-machine/README.md [hunk 1]
```diff
@@ -1,6 +1,19 @@
...
+
+## Diagram
+
+[![](https://mermaid.ink/img/pako:eNqdlD1v3DAMhv8KoaFLLujuAlmapUOmQzYtrEWficqSQ9FJDkH-e2UrV-sa5wPxYlt8H5J4TfPJtNGRaUyiu4lCS9eMB8HBhhFFueURg8JtIrHBBsjXT8-Ujy6vri72JPckDcjMJgUOrIwekqJSURfJor5Gxb1GoRVw-ajo_gX_S6yThEq2pittNOc1AYODjiWnTkrjq4ZP0DS6WX37Cx5Y-zOiMD7GEQ4RtJc4HfollkpovmY_6ib25KlVwFY5BvieE3oPXZRhRTZsG6LjjltcoBGPPqJbgcq5E3CPnpfGa3IlfgvhH-Bu_R4JOmSfTmBW_1jldZHanBfTi6HfgESiwEAp4YHO6Q1nHafR43Fb_wXb3rAu0MMH9lHY9nJjCrfdfHciC0KuGs3t8TwTLuMZ6LGezs9N6AYUYo7HuVxhdrD2OBJqvmHb58raE1jT5R8l5VdrqjyLSWZnBpIB2eU18DQHrMnMQNY0-dFRh5NXa2x4zlKcNO6PoTWNykQ7Uxp92Rqm6dCnfEqOs3U3ZbUsG-b5L4HncGU?type=png)](https://mermaid.live/edit#pako:eNqdlD1v3DAMhv8KoaFLLujuAlmapUOmQzYtrEWficqSQ9FJDkH-e2UrV-sa5wPxYlt8H5J4TfPJtNGRaUyiu4lCS9eMB8HBhhFFueURg8JtIrHBBsjXT8-Ujy6vri72JPckDcjMJgUOrIwekqJSURfJor5Gxb1GoRVw-ajo_gX_S6yThEq2pittNOc1AYODjiWnTkrjq4ZP0DS6WX37Cx5Y-zOiMD7GEQ4RtJc4HfollkpovmY_6ib25KlVwFY5BvieE3oPXZRhRTZsG6LjjltcoBGPPqJbgcq5E3CPnpfGa3IlfgvhH-Bu_R4JOmSfTmBW_1jldZHanBfTi6HfgESiwEAp4YHO6Q1nHafR43Fb_wXb3rAu0MMH9lHY9nJjCrfdfHciC0KuGs3t8TwTLuMZ6LGezs9N6AYUYo7HuVxhdrD2OBJqvmHb58raE1jT5R8l5VdrqjyLSWZnBpIB2eU18DQHrMnMQNY0-dFRh5NXa2x4zlKcNO6PoTWNykQ7Uxp92Rqm6dCnfEqOs3U3ZbUsG-b5L4HncGU)
```
Comment (codingWithRach) [inline libs/state-machine/README.md:14]: minor typo on the diagram: "**is** fails validation"

**Review by codingWithRach — APPROVED** [review]
This is brilliant - so much better than anything I could have written!

**Comment by codingWithRach** [comment]
I've just spotted that apiDeploymentSettings.ts states that 
`In development, variables can be adjusted locally on the command line e.g.
  	NEWSLETTERS_UI_SERVE=false npm run dev`
 This comment should probably point people towards the `.env.local` file

**Comment by dblatcher** [comment]
> apiDeploymentSettings

Good point - either method would work, so I'll change the readme and the comment to mention both.

File: libs/state-machine/README.md (hunk 1 above)
Comment (dblatcher) [reply libs/state-machine/README.md:14]: updating! glad I save the link to the editor :)

File: apps/newsletters-api/README.md [hunk 2]
```diff
@@ -9,9 +9,12 @@ On the default configuration, the API serves a the UI on its index page, so http
 
 ### Configuring the local instance with environment variables
 
-Copy **apps/newsletters-api/env.local.example.txt** and rename it **.env.local**. This file will be .gitignored.
+Local environment variables can be set by:
 
-By default, the local application uses an in-memory storage system (so data you enter will not persist when the application restarts). To use an S3 bucket for storage instead, you will need to set the s3 parameters in **.env.local**. You will need credentials for the right AWS account (IE the account the S3 bucket it in) from [Janus](https://janus.gutools.co.uk/), for the API to be able to access the bucket and read/write files.
+-   Coping **apps/newsletters-api/env.local.example.txt** and rename it **.env.local** (This file will be .gitignored), or
```
Comment (codingWithRach) [inline apps/newsletters-api/README.md:14]: Minor typo: Copying

**Review by codingWithRach — APPROVED** [review]

**Pull request merged**